beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,486 @@ title: Quick Start
 
-<Sandpack>
-
-```js App.js
-import { people } from './data.js';
-import Profile from './Profile.js';
+In React, there is no special syntax for writing conditions. Instead, you'll use the same techniques as you use when writing regular JavaScript code. To render a component conditionally, you can use [`if`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else) statement:
 
-export default function Gallery() {
-  return (
-    <section>
-      <h1>Amazing scientists</h1>
-      {people.map(person => (
-        <Profile
-          key={person.id}
-          name={person.name}
-          imageId={person.imageId}
-        />
-      ))}
-    </section>
-  );
-}
-```
-
-```js Profile.js
-export default function Profile({ name, imageId }) {
-  const imageUrl = (
-    'https://i.imgur.com/' +
-    imageId +
-    's.jpg'
-  );
-  return (
-    <img
-      className="avatar"
-      src={imageUrl}
-      alt={name}
-    />
-  );
+```js
+let content;
+if (isLoggedIn) {
+  content = <AdminPanel />;
+} else {
+  content = <LoginForm />;
 }
+<div>
+  {content}
+</div>
 ```
 
-```js data.js
-export const people = [{
-  id: 0,
-  name: 'Creola Katherine Johnson',
-  imageId: 'MK3eW3A'
-}, {
-  id: 1,
-  name: 'Mario José Molina-Pasquel Henríquez',
-  imageId: 'mynHUSa'
-}, {
-  id: 2,
-  name: 'Mohammad Abdus Salam',
-  imageId: 'bE7W1ji'
-}, {
-  id: 3,
-  name: 'Percy Lavon Julian',
-  imageId: 'IOjWm71'
-}, {
-  id: 4,
-  name: 'Subrahmanyan Chandrasekhar',
-  imageId: 'lrWQx8l'
-}];
-```
+Or, if you prefer terser code, you can use the [conditional `?` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator):
 
-```css
-img { margin: 0 10px 10px 0; }
-.avatar { border-radius: 50%; }
+```js
+<div>
+  {isLoggedIn ? (
+    <AdminPanel />
+  ) : (
+    <LoginForm />
+  )}
+</div>
 ```
 
-</Sandpack>
+If you don't have the `else` branch, you can also use a shorter [logical `&&` syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#short-circuit_evaluation):
 
-<LearnMore path="/learn/describing-the-ui">
+```js
+<div>
+  {isLoggedIn && <AdminPanel />}
+</div>
+```
 
-Read **[Describing the UI](/learn/describing-the-ui)** to learn how to make things appear on the screen, including declaring components, importing them, writing JSX with the curly braces, and writing conditions and lists.
+If you're unfamiliar with this JavaScript syntax, you can start by always using `if...else`.
 
-</LearnMore>
+## Rendering lists {/*rendering-lists*/}
 
-### Chapter 2 overview: Adding interactivity {/*chapter-2-overview-adding-interactivity*/}
+You will rely on JavaScript features like [`for` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for) and the [array `map()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to render lists of components.
```

Comment:
```suggestion
You will rely on JavaScript features like [`for` loops](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for) and the [array `map()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to render lists of components.
```

Even worth mentioning `for` loops here? I can't think of the last time I've used one for list rendering.

Reply:
I think it's worth mentioning just to dispel the notion there's something special about map() here.